src/libs/actions/Report.js

```diff
@@ -1388,9 +1388,11 @@ function updateWriteCapabilityAndNavigate(report, newValue) {
 
 /**
  * Navigates to the 1:1 report with Concierge
+ *
+ * @param {Boolean} ignoreConciergeReportID
```

Comment:
Can you ask a comment to the parameter what and why this is used for? If not here, before the if condition.

Reply:
Sure. Thanks for the review. I will add comments.

Reply:
I have added comments and requested a re-review.
Resolving this conversation for now.